server/src/sim/makeccocs.py

```diff
@@ -19,11 +19,12 @@
 
 ## Set defaults for testing makeccocs
 default_progname = 'SBCC'
-default_ccparams = [] # [0.9, 0.28, 154000.0, 1.0]
-default_ccplot = [] # [2e6, [0, []]]
-default_coparams = [] # [0.3, 0.5, 0.7, 0.9] 
```

Comment:
@robynstuart let's not assume 'nan' in anything that potentially comes from frontend.